dt-assets/functions/enqueue-scripts.php

```diff
@@ -110,13 +110,13 @@ function dt_site_scripts() {
             'template_dir' => get_template_directory_uri(),
             'translations' => [
                 'days_of_the_week' => [
-                    _x( "Su", 'Dates', 'disciple_tools' ),
-                    _x( "Mo", 'Dates', 'disciple_tools' ),
-                    _x( "Tu", 'Dates', 'disciple_tools' ),
-                    _x( "We", 'Dates', 'disciple_tools' ),
-                    _x( "Th", 'Dates', 'disciple_tools' ),
-                    _x( "Fr", 'Dates', 'disciple_tools' ),
-                    _x( "Sa", 'Dates', 'disciple_tools' )
+                    _x( "Su", 'Abbreviation of Sunday', 'disciple_tools' ),
```

Comment:
This will trigger a re-translation of each of these. I don't think it is worth it.

Reply:
I disagree. It is only 7 strings and without a clearer context, they cause confusion.